packages/@react-spectrum/tag/src/TagGroup.tsx

```diff
@@ -44,7 +46,8 @@ function TagGroup<T extends object>(props: SpectrumTagGroupProps<T>, ref: DOMRef
     children,
     actionLabel,
     onAction,
-    labelPosition
+    labelPosition,
+    renderEmptyState = () => <div>{stringFormatter.format('noTags')}</div>
```

Comment:
Should we make this div the same height as a Tag? Right now the height of the container will shrink when the final tag is removed and replaced with this div.